FakeDetector.py

```diff
@@ -64,6 +75,7 @@
 POSTPROCESS_ERROR=94
 
 mediaExtensions = ['.mkv', '.avi', '.divx', '.xvid', '.mov', '.wmv', '.mp4', '.mpg', '.mpeg', '.vob', '.iso', '.m4v']
+bannedMediaExtensions = movies_format=os.environ['NZBPO_BANNEDEXTENSIONS'].replace(' ', '').split(',')
```

Comment:
1. The part `movies_format=` looks superfluous. 

2. Instead of `os.environ['NZBPO_BANNEDEXTENSIONS']` use `os.environ.get('NZBPO_BANNEDEXTENSIONS')`. The former causes the script to fail if the option isn't set. That may happen if user updates from an older version of the script and haven't saved the settings in nzbget after update yet.